code/modules/research/designs/guncrafting_designs.dm

```diff
@@ -0,0 +1,544 @@
+	id = "base_revolver_38"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 2000)
+	build_path = /obj/item/weapon/gun_attachment/base/revolver_38
+	category = list("Revolver Parts")
+
+/datum/design/base_revolver_357
+	name = "Revolver Base: .357 Revolver"
+	desc = "A revolver base."
+	id = "base_revolver_357"
+	req_tech = list("combat" = 1, "materials" = 1, "syndicate" = 5)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 2000)
+	build_path = /obj/item/weapon/gun_attachment/base/revolver_357
+	category = list("Revolver Parts")
+
+/datum/design/base_shotgun
+	name = "Gun Ballistic Base: Shotgun"
+	desc = "Security, Antag, I'm the guy with the shotgun."
+	id = "base_shotgun"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 4000)
+	build_path = /obj/item/weapon/gun_attachment/base/shotgun
+	category = list("Guncrafting Parts")
+
+/datum/design/ebullet_disorient
+	name = "Gun Energy Modifier: Disorienter"
+	desc = "A disorienter mod."
+	id = "ebullet_disorient"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/disorienter
+	category = list("Guncrafting Parts")
+
+/datum/design/ebullet_decloner
+	name = "Gun Energy Modifier: Decloner"
+	desc = "A decloner mod."
+	id = "ebullet_decloner"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/decloner
+	category = list("Guncrafting Parts")
+
+/datum/design/ebullet_speed
+	name = "Gun Energy Modifier: Speed"
+	desc = "A speed mod."
+	id = "ebullet_speed"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/speed
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_rad
+	name = "Gun Bullet Modifier: Polonium"
+	desc = "A polonium mod."
+	id = "bullet_rad"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/gun_attachment/bullet/polonium
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_fire
+	name = "Gun Bullet Modifier: Incendiary"
+	desc = "A fire mod."
+	id = "bullet_fire"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/bullet/fire
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_bleed
+	name = "Gun Bullet Modifier: Haemorrhage-Inflicting"
+	desc = "A bleeding mod."
+	id = "bullet_bleed"
+	req_tech = list("combat" = 1, "materials" = 1)
```

Comment:
Mmmm, I would say this one needs some research to get. Perhaps combat 3, mat 4, and bio 3.